base_menu_visibility_restriction/models/ir_ui_menu.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models, tools
+
+
+class IrUiMenu(models.Model):
+
+    _inherit = "ir.ui.menu"
+
+    excluded_groups_id = fields.Many2many(
```

Comment:
Field name is not correct according guidelines:
```suggestion
    excluded_group_ids = fields.Many2many(
```
Maybe you have selected seeing other existing fields that are incorrect in these models, but we shouldn't extend that bad names on new ones.